api/pkg/utils/obs/log.go

```diff
@@ -35,10 +34,9 @@ const (
 	defaultLogLevel        = "info"
 	unknownHost            = "unknownhost"
 	unknownUser            = "unknownuser"
-	configFileName         = "log.conf"
+	configFileName         = "/etc/log.conf"
```

Comment:
change it to /etc/multi-cloud/multi-cloud.conf

Reply:
Done